### DISCUSSION TITLE
Strapi v5

### DISCUSSION THREAD

**Discussion by m7rlin** [magictm/strapi-plugin-deep-populate#1] [Announcements]
## Strapi v5

Post here your ideas for this plugin in strapi v5.

For now I won't develop this plugin for Strapi v5. I will wait for a stable v5 release.

If you don't want to wait just fork it and create new branch "strapi-v5" and update the plugin there it will help me then update the plugin :)

Thank you for your understanding

**Comment by cezarvrabie99** [comment]
Hello! Following [this PR](https://github.com/strapi/strapi/pull/21034) that was released in strapi v5.0.0-rc.13, the plugin is now broken. 
Here the team added validation for the populate query param. Even though this plugin does not officially support v5, it used to work ok before this.

**Reply by m7rlin** [reply]
I will update to strapi v5 after some time after stable release.

**Comment by zhawzid-ottj** [comment]
This is a (probably) working and slightly optimized code for strapi v5
https://github.com/zhawzid-ottj/strapi-v5-plugin-deep-populate
you're gonna need a different request param deepPopulate
